src/napari/_qt/_tests/test_viewer_qt_integration.py

```diff
@@ -35,6 +38,8 @@ def test_drop_python_file(make_napari_viewer, tmp_path):
     # Check that the file was executed
     assert viewer.layers[0].name == 'Dropped Image'
 
+    qapp_mock.assert_not_called()  # Ensure napari.run is early stopped
```

Comment:
```suggestion
    # Ensure that napari.run is stopped early and event loops are not running
    qapp_mock.assert_not_called()
```

Reply:
It would help me to have a better comment to understand this assertion and its purpose.